Cannot find module 'chalk'

In a consuming add-on, an error is displayed:

```
Error: Cannot find module 'chalk'
    at Function.Module._resolveFilename (module.js:470:15)
    at Function.Module._load (module.js:418:25)
    at Module.require (module.js:498:17)
    at require (internal/module.js:20:19)
    at Object.<anonymous> (/Users/Micarrol/Devel/ember-frost-date-picker/node_modules/ember-test-utils/cli/lint-all-the-things.js:3:15)
    at Module._compile (module.js:571:32)
    at Object.Module._extensions..js (module.js:580:10)
    at Module.load (module.js:488:32)
    at tryModuleLoad (module.js:447:12)
    at Function.Module._load (module.js:439:3)
```

--- a/package.json
+++ b/package.json
@@ -59,6 +59,7 @@
     "ember-addon"
   ],
   "dependencies": {
+    "chalk": "^1.1.3",
     "dockerfile_lint": "^0.2.6",
     "ember-cli-babel": "^5.1.5",
     "ember-template-lint": "^0.6.3",

